.github/workflows/README.md

```diff
@@ -714,6 +715,78 @@ This workflow is built from multiple composite actions listed below:
 - [java-gradle-test](https://github.com/bakdata/ci-templates/tree/main/actions/java-gradle-test)
+- [java-gradle-assess-code-quality](https://github.com/bakdata/ci-templates/tree/main/actions/java-gradle-assess-code-quality)
+
+### Input Parameters
+
+| Name               | Required | Default Value |  Type   | Description                                                                                                   |
+| ------------------ | :------: | :-----------: | :-----: | ------------------------------------------------------------------------------------------------------------- |
+| java-distribution  |    ❌    |   microsoft   | string  | [Java distribution](https://github.com/actions/setup-java#supported-distributions) to be installed            |
+| java-version       |    ❌    |      11       | string  | Java version to be installed                                                                                  |
+| gradle-version     |    ❌    |    wrapper    | string  | [Gradle version](https://github.com/gradle/gradle-build-action#use-a-specific-gradle-version) to be installed |
+| gradle-cache       |    ❌    |     true      | boolean | Whether Gradle caching is enabled or not                                                                      |
+| working-directory  |    ❌    |       .       | string  | Working directory of your Gradle artifacts                                                                    |
+| download-lfs-files |    ❌    |    "false"    | string  | Whether the Git checkout action should resolve LFS files or not                                               |
+
+### Secret Parameters
+
+For Sonarcloud you need to provide a `sonar-token` and a `sonar-organization` to publish code quality results. In case of Sonatype, the action
+requires you to have a `signing-secret-key-ring` (base64 encoded), a `signing-key-id` and a `signing-password` to sign
+your build artifacts and additionally an `ossrh-username` and an `ossrh-password` to publish the signed artifacts to
+Nexus.
+
+| Name                    | Required | Description                                                    |
+| ----------------------- | :------: | -------------------------------------------------------------- |
+| sonar-token             |    ✅    | Token for Sonarcloud                                           |
+| sonar-organization      |    ✅    | Organization for Sonarcloud                                    |
+| signing-secret-key-ring |    ✅    | Key ring (base64 encoded) for signing the Sonatype publication |
+| signing-key-id          |    ✅    | Key id for signing the Sonatype publication                    |
+| signing-password        |    ✅    | Password for signing the Sonatype publication                  |
+| ossrh-username          |    ✅    | Username for signing into Sonatype repository                  |
+| ossrh-password          |    ✅    | Password for signing into Sonatype repository                  |
```

Comment:
These do not seem to be a secrets that can be set currently for the `java-gradle-base` workflow

Reply:
https://github.com/bakdata/ci-templates/blob/feat/split-test/.github/workflows/java-gradle-base.yaml#L98

Reply:
Yes, but where are the `ossrh-username` and `ossrh-password` marked above?